rosidl_runtime_c/QUALITY_DECLARATION.md

```diff
@@ -116,7 +116,8 @@ A description of how coverage statistics are summarized from this page, can be f
 
 ### Performance [4.iv]
 
-`rosidl_runtime_c` does not currently have performance tests.
+Most of the features provided by this package are declarations of types and functions, and therefore do not require testing.
+The BoundedVector class is benchmarked and the most recent test results can be found [here](http://build.ros2.org/view/Rci/job/Rci__benchmark_ubuntu_focal_amd64/BenchmarkTable/).
```

Comment:
You'll also need to fix the QL level of rosidl_typesupport_interface in the dependencies section